src/main/java/MassimoBoi/MassimoBoi.java

```diff
@@ -0,0 +1,205 @@
+package MassimoBoi;
+
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import MassimoBoiException.*;
+
+public class MassimoBoi {
+    public static void main(String[] args) throws IOException {
+
+        String logo = " __  __               _                   _           _ \n" +
+                "|  \\/  | __ _ ___ ___(_)_ __ ___   ___   | |__   ___ (_)\n" +
+                "| |\\/| |/ _` / __/ __| | '_ ` _ \\ / _ \\  | '_ \\ / _ \\| |\n" +
+                "| |  | | (_| \\__ \\__ \\ | | | | | | (_) | | |_) | (_) | |\n" +
+                "|_|  |_|\\__,_|___/___/_|_| |_| |_|\\___/  |_.__/ \\___/|_|";
+
+        System.out.println("Hello from\n" + logo);
+        Scanner myObj = new Scanner(System.in);
+        String userInput = "";
+        List<Task> taskList = new ArrayList<>();
+        try{
+            FileClass.loadList(taskList);
+        } catch(FileNotFoundException e){
+
+        };
+
+
+        greet();
+        makeHorizontalRow();
+
+        /*
+         * The while loop takes in user input and processes them according to different preset commands
+         * List of user inputs: task type, mark and unmark task, list, bye (to end chat)
+         */
+        while (true) {
+            userInput = myObj.nextLine();
+            if (userInput.equals("bye")) {
+                goodbye(taskList);
+                break;
+            }
+            try{
+                handleInput(userInput, taskList);
+            } catch(UnknownCommandType e){
+                e.errorMessage();
+                greet();
+            } catch(EmptyToDo e){
+                e.errorMessage();
+            } catch(NoDueDate e){
+                e.errorMessage();
+            } catch (EmptyDeadline e){
+                e.errorMessage();
+            } catch(MassimoBoiException e){
+                e.errorMessage();
+            } catch(IndexOutOfBoundsException e){
+
+                System.out.println("The task number you have tried to mark, unmark, or delete does not exist");
+            } catch(IOException e){
+                System.out.println("The task number you have tried to mark/unmark does not exist");
+
+
+                System.out.println("The task number you have tried to mark/unmark does not exist");
+
+            }
+            finally {
+                makeHorizontalRow();
+            }
+
+
+
```

Comment:
why is there suddenly 3 lines of space here? please check through your code for other such large spaces.